test/helper-functions.bats

```diff
@@ -0,0 +1,43 @@
+load 'test_helper/common'
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function teardown() {
+    run_teardown_file_if_necessary
+}
+
+function setup_file() {
+    local PRIVATE_CONFIG
+    PRIVATE_CONFIG="$(duplicate_config_for_container .)"
+    docker run -d --name mail_helper_functions \
+		-v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
+		-v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
+		-e ENABLE_FETCHMAIL=1 \
+		--cap-add=NET_ADMIN \
+		-e DMS_DEBUG=0 \
+		-h mail.my-domain.com -t "${NAME}"
+    wait_for_finished_setup_in_container mail_helper_functions
+}
```

Comment:
Seemed a bit weird having `--cap-add` between the two env options. Main concern was mixed indentation (spaces and tabs), fixed:

```suggestion
function setup_file() {
    local PRIVATE_CONFIG
    PRIVATE_CONFIG="$(duplicate_config_for_container .)"
    docker run -d --name mail_helper_functions \
        --cap-add=NET_ADMIN \
        -v "${PRIVATE_CONFIG}":/tmp/docker-mailserver \
        -v "$(pwd)/test/test-files":/tmp/docker-mailserver-test:ro \
        -e ENABLE_FETCHMAIL=1 \
        -e DMS_DEBUG=0 \
        -h mail.my-domain.com -t "${NAME}"
    wait_for_finished_setup_in_container mail_helper_functions
}
```